en/docs/SetupAndInstall/UpgradingWSO2APIManager/upgrading-from-260-to-300.md

```diff
@@ -1,32 +1,316 @@
+        /
+        UPDATE REG_RESOURCE_RATING SET REG_RESOURCE_RATING.REG_RESOURCE_NAME=(SELECT REG_RESOURCE.REG_NAME FROM REG_RESOURCE WHERE REG_RESOURCE.REG_VERSION=REG_RESOURCE_RATING.REG_VERSION)
+        /
+        ```
+        
+        ```tab="PostgreSQL"
+        -- Update the REG_PATH_ID column mapped with the REG_RESOURCE table --
+        UPDATE REG_RESOURCE_TAG SET REG_PATH_ID=(SELECT REG_PATH_ID FROM REG_RESOURCE WHERE REG_RESOURCE.REG_VERSION=REG_RESOURCE_TAG.REG_VERSION);
+        
+        UPDATE REG_RESOURCE_COMMENT SET REG_PATH_ID=(SELECT REG_PATH_ID FROM REG_RESOURCE WHERE REG_RESOURCE.REG_VERSION=REG_RESOURCE_COMMENT.REG_VERSION);
+        
+        UPDATE REG_RESOURCE_PROPERTY SET REG_PATH_ID=(SELECT REG_PATH_ID FROM REG_RESOURCE WHERE REG_RESOURCE.REG_VERSION=REG_RESOURCE_PROPERTY.REG_VERSION);
+        
+        UPDATE REG_RESOURCE_RATING SET REG_PATH_ID=(SELECT REG_PATH_ID FROM REG_RESOURCE WHERE REG_RESOURCE.REG_VERSION=REG_RESOURCE_RATING.REG_VERSION);
+        
+        -- Delete versioned tags, were the PATH_ID will be null for older versions --
+        delete from REG_RESOURCE_PROPERTY where REG_PATH_ID is NULL;
+        
+        delete from REG_RESOURCE_RATING where REG_PATH_ID is NULL;
+        
+        delete from REG_RESOURCE_TAG where REG_PATH_ID is NULL;
+        
+        delete from REG_RESOURCE_COMMENT where REG_PATH_ID is NULL;
+        
+        delete from REG_PROPERTY where REG_ID NOT IN (select REG_PROPERTY_ID from REG_RESOURCE_PROPERTY);
+        
+        delete from REG_TAG where REG_ID NOT IN (select REG_TAG_ID from REG_RESOURCE_TAG);
+        
+        delete from REG_COMMENT where REG_ID NOT IN (select REG_COMMENT_ID from REG_RESOURCE_COMMENT);
+        
+        delete from REG_RATING where REG_ID NOT IN (select REG_RATING_ID from REG_RESOURCE_RATING);
+        
+        -- Update the REG_PATH_NAME column mapped with the REG_RESOURCE table --
+        UPDATE REG_RESOURCE_TAG SET REG_RESOURCE_NAME=(SELECT REG_NAME FROM REG_RESOURCE WHERE REG_RESOURCE.REG_VERSION=REG_RESOURCE_TAG.REG_VERSION);
+        
+        UPDATE REG_RESOURCE_PROPERTY SET REG_RESOURCE_NAME=(SELECT REG_NAME FROM REG_RESOURCE WHERE REG_RESOURCE.REG_VERSION=REG_RESOURCE_PROPERTY.REG_VERSION);
+        
+        UPDATE REG_RESOURCE_COMMENT SET REG_RESOURCE_NAME=(SELECT REG_NAME FROM REG_RESOURCE WHERE REG_RESOURCE.REG_VERSION=REG_RESOURCE_COMMENT.REG_VERSION);
+        
+        UPDATE REG_RESOURCE_RATING SET REG_RESOURCE_NAME=(SELECT REG_NAME FROM REG_RESOURCE WHERE REG_RESOURCE.REG_VERSION=REG_RESOURCE_RATING.REG_VERSION);
+        ```
+    
+!!! warning "Not recommended"
+    If you decide to proceed with registry resource versioning enabled, Add the following configuration to the `deployment.toml` file of new WSO2 API Manager. 
+    `<NEW_API-M_HOME>/repository/conf/deployment.toml`
+    
+    ```
+    [registory.static_configuration]
+    enable=false
```

Comment:
We need to have 
``` suggestion
    enable=true
```
 to enable the registry versioning. Shall we change it?

Reply:
Changed